extensions/sql-migration/src/constants/strings.ts

```diff
@@ -1289,3 +1290,113 @@ export const SQLDB_COL_COPY_DURATION = localize('sql.migration.sqldb.column.copy
+export const TDE_WIZARD_MSG_MANUAL = localize('sql.migration.tde.msg.manual', "You have choosen to manually migrate certificates.");
+export const TDE_WIZARD_MSG_TDE = localize('sql.migration.tde.msg.tde', "You have given access to ADS to migrate the encryption certificates and database.");
+export const TDE_WIZARD_MSG_EMPTY = localize('sql.migration.tde.msg.empty', "No encrypted database selected.");
+export const TDE_RULE_ID = localize('sql.migration.tde.rule.id', "TdeEnabled");
+
+export function TDE_MIGRATION_ERROR(message: string): string {
+	return localize('sql.migration.starting.migration.error', "An error occurred while starting the certificate migration: '{0}'", message);
+}
+
+export function TDE_MIGRATION_ERROR_DB(name: string, message: string): string {
+	return localize('sql.migration.starting.migration.dberror', "Error migrating certificate for database {0}. {1}", name, message);
+}
+
+export function TDE_MSG_DATABASES_SELECTED(selected: number, message: string): string {
+	return localize('sql.migration.tde.msg.databases.selected', "{0} TDE databases selected for migration. {1}", selected, message);
+}
+
+export function TDE_WIZARD_DATABASES_SELECTED(encryptedCount: number, totalCount: number): string {
+	return localize('sql.migration.tde.wizard.databases.selected', "{0} out of {1} selected database is using transparent data encryption.", encryptedCount, totalCount);
+}
+
+
+export const TDE_WIZARD_MIGRATION_OPTION_MANUAL_WARNING = localize('sql.migration.tde.wizard.optionmanual.warning', "You must migrate the certificates before proceeding with the migration otherwise the migration will fail. {0}.");
+
+export const TDE_WIZARD_ADS_CERTS_INFO = localize('sql.migration.network.share.header.text', "Please enter a location where the SQL server will export the certificates. Please ensure the SQL server service has access access to this path. The current user should have administrator privileges on the computer where this network path is.");
+
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_LABEL = localize('sql.migration.tde.wizard.network.share.label', "Network path for certificate");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_PLACEHOLDER = localize('sql.migration.tde.wizard.network.share.placeholder', "Enter network path");
+export const TDE_WIZARD_CERTS_NETWORK_SHARE_INFO = localize('sql.migration.tde.wizard.network.share.info', "Network path where certificate will be placed.");
+
+export const TDE_WIZARD_CERTS_DOMAIN_LABEL = localize('sql.migration.tde.wizard.domain.label', "Domain");
+export const TDE_WIZARD_CERTS_DOMAIN_PLACEHOLDER = localize('sql.migration.tde.wizard.domain.placeholder', "Enter Domain");
+export const TDE_WIZARD_CERTS_DOMAIN_INFO = localize('sql.migration.tde.wizard.domain.info', "Domain for the credentials provided.");
+
+export const TDE_WIZARD_CERTS_USERNAME_LABEL = localize('sql.migration.tde.wizard.username.label', "Username");
```

Comment:
I believe "Username" is already a string used elsewhere in this file - since these strings are localized, might be worth removing any duplicates and reuse any already-existing ones

Reply:
Same with a lot of the below as well, e.g. succeeded, failed, etc

Reply:
Removed those. Updated the names for state, so it can be reuse by others too.